src/main.rs

```diff
@@ -110,8 +119,10 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                     .cookie_secure(config.production)
                     .cookie_name(SESSION_COOKIE_NAME.to_string())
                     .session_lifecycle(
-                        PersistentSession::default()
-                            .session_ttl(Duration::seconds(config.get_session_ttl())),
+                        PersistentSession::default().session_ttl(Duration::seconds(
+                            i64::try_from(config.get_session_ttl())
```

Comment:
Brrr, why do we still have `i64`? Can you please make a ticket to fix this as well? It really has no reason to exist, as it can never be negative.

Reply:
I'm not sure how, `PersistentSession::default().session_ttl(..)` requires an `actix_web::cookie::time::Duration` and the function `Duration::seconds` requires an i64, hence the casting.
```rust
pub const fn seconds(seconds: i64) -> Self {
    Self::new_ranged_unchecked(seconds, Nanoseconds::new_static::<0>())
}
```

Reply:
Ok then, ignore this then.